packages/shared-lib/src/package-manager/package.ts

```diff
@@ -172,6 +177,78 @@ export namespace ExpectedPackage {
 			}
 		}[]
 	}
+	export interface ExpectedPackageHtmlTemplate extends Base {
+		type: PackageType.HTML_TEMPLATE
+		content: {
+			/** path to the HTML template */
+			path: string
+			/** Add prefix to output artifacts */
+			outputPrefix: string
+		}
+		version: {
+			renderer?: {
+				/** Renderer width, defaults to 1920 */
+				width?: number
+				/** Renderer width, defaults to 1080 */
+				height?: number
+				/** Zoom level, defaults to 1 */
+				zoom?: number
+				/** (defaults to black) */
+				backgroundColor?: string
```

Comment:
How is this color specified? CSS Colors? Can this be set to "transparent"? Can this be `background` and not `backgroundColor` so that this is set to a `background-image` of choice?